src/services/copyConfig/rba/rbaOptions.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright: Copyright 2023 SAP SE or an SAP affiliate company and cdc-tools-chrome-extension contributors
+ * License: Apache-2.0
+ */
+
+import Options from '../options.js'
+
+export default class RbaOptions extends Options {
+  static ACCOUNT_TAKEOVER_PROTECTION = 'accountTakeoverProtection'
+  static UNKNOWN_LOCATION_NOTIFICATION = 'unknownLocationNotification'
+  static RULES = 'rules'
+  #rba
+
+  constructor(rba) {
+    super({
+      id: 'rba',
+      name: 'rba',
```

Comment:
Instead of "Rba", the UI it should show "Risk Based Authentication", so name can be "riskBasedAuthentication".